docs/advanced/dcrtime.md

```diff
@@ -0,0 +1,105 @@
+
+```
+$ dcrtime --testnet -v
+cd90cc268d9ceef6e276bfa7a615c5f85b5a27b0d917ee3f1f1e5d5598f2fa00
+cd90cc268d9ceef6e276bfa7a615c5f85b5a27b0d917ee3f1f1e5d5598f2fa00 Verify
+cd90cc268d9ceef6e276bfa7a615c5f85b5a27b0d917ee3f1f1e5d5598f2fa00 Not anchored
+```
+
+If we wait until the transaction containing the merkle root has been mined into a block and call dcrtime again, the dcrtime server will verify that the transaction has been anchored, as well as provide the merkle path (path to the merkle root) {SB: accurate?}, the transaction ID (TxID) of the transaction containing the merkle root, and a `Chain Timestamp` providing the time the block was mined. 
+
+```
+$ dcrtime --testnet -v 8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6
+8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6 Verify
+8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6 OK
+  Chain Timestamp: 1497013614
+  Merkle Root    : 8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6
+  TxID           : 4172a560a7035c169c4da60cba2cb1fbac686bd01224e09a1a56ce5e6f31cff0
+```
+
+Note that the dcrtime client does verify the merkle path that the server returns. 
+
+Our file is now anchored in the Decred blockchain. Any third party can now prove that `myfile.txt` existed, in its exact form, at the time the block was mined (Chain Timestamp) {SB: is the 'Cellection timestamp relevant here?'}. This can be proved by taking our original `myfile.txt` file, calculating its merkle root, and comparing it to the merkle root stored in the transaction submitted by dcrtime (which we can look up via the TxID in the [block exporer](https://explorer.dcrdata.org/)).
+
+### Politeia
+
+Building on our simple example, the following example illustrates how dcrtime is used to timestamp data in Politeia.
+
+Each hour, the Politeia server commits all data uploaded to it (proposals, votes, comments, etc.) to a public git [repo](https://github.com/decred-proposals/mainnet). It then creates {SB: sequence correct?} a hash of each commit, timestamps each commit hash using dcrtime, and anchors the hashes on the Decred blockchain. Once the transaction containing the merkle root of the hash is mined, the corresponding merkle path and transaction hash is stored in the proposal repository. This allows all commits to the repo to be verified against a copy of the Decred blockchain. {SB: It would be nice to understand this better myself, and possibly put in the docs. Specifically, is there just one hash (the hash of a commit that contains all the changes)? Is each piece of data hashed and timestamped separately? Assuming each piece of data has its own merkle root in a larger tree (what is put into the transaction), how does the user traverse the tree to find the merkel root they're interested in?}
```

Comment:
This is incorrect. Pi commits into git as needed (edits, new props etc) and before anchoring it flushes all comments and vote journals. once git has stabilized the last commit is anchored + all other commit hashes up to the prior anchor.

Reply:
Went back and rewrote this section to reflect above, as well as fleshing out the example.

Reply:
Each git commit has an implict hash. We anchor those hashes.

Reply:
Not sure exactly what an implicit hash is, even after a few minutes of googling...Have changed the text to this. Does this work? 

>...These journal files are then committed to the repo when other data such as a new proposal is submitted, or every hour if no other data is submitted. Once Politeia verifies that these commits are in the repo, dcrtime is used to anchor the implicit hash for each commit. For each hash anchored, an anchor file is committed to the `/anchors` directory in the repo. To fasciliate verification, metadata for each anchor is also appended to a single text file, `anchor_audit_trail.txt`, which provides a chonological record of anchors.